docs/source/cookbook/aggregated_node.rst

```diff
@@ -0,0 +1,89 @@
+Aggregated nodes
+----------------
+
+The `AggregatedNode` is a special node that allows constraints to be added to a model that apply to a group of (possibly disconnected) nodes, enforcing either a fixed ratio or fixed maximum.
```

Comment:
`.... enforcing either a fixed ratio of flow and/or a total maximum flow.`

Is this more accurate? I assume the solver supports minimum flow too?